### PR TITLE
feat: e2e/smoke tests

### PR DESCRIPTION
**Pull request by peterstarling**
See: https://trello.com/c/FMU3iWrH/933-create-e2e-smoke-tests

### CODE REVIEW

File: apps/e2e-tests/config.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const config = {
+  appOrigin: process.env.APP_ORIGIN || 'https://api-dev.hub.asap.science',
```
Comment (jeysal) [inline apps/e2e-tests/config.ts:2]: I think app origin so far usually refers to the frontend, we probably have some other environment variables in use for the api origin?

File: apps/e2e-tests/jest-e2e.config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  basename(__dirname),
+);
+
+module.exports = {
+  ...base,
+
+  rootDir: __dirname,
+  displayName: 'e2e-tests',
+
+  testMatch: ["<rootDir>/tests/**/*.test.ts"],
```
Comment (jeysal) [inline apps/e2e-tests/jest-e2e.config.js:16]: I think the Jest default pattern should be sufficient?

**Review by jeysal — COMMENTED** [review]
Yeah this is about what I expected, I guess the interesting bit you'll tackle next will be CI integration i.e. running this on feature envs / on dev to prevent prod deploy on error / on prod to highlight any error?

File: apps/e2e-tests/config.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export const config = {
+  appOrigin: process.env.ASAP_API_URL || 'https://api-dev.hub.asap.science',
```
Comment (fampinheiro) [inline apps/e2e-tests/config.ts:2]: We use app origin to mention the main page.
To avoid expectation mismatch I would change this to `apiOrigin`.

**Review by fampinheiro — APPROVED** [review]

**Review by fampinheiro — APPROVED** [review]

**Pull request merged**